README.md

```diff
@@ -2,9 +2,59 @@
 
 Create Ruby clients from a protocol buffer description of an API.
 
-**Note** This project is a preview. Please try it out and let us know what you think, 
+**Note** This project is a preview. Please try it out and let us know what you think,
 but there are currently no guarantees of stability or support.
 
+## Usage
+### Install the Proto Compiler
+This generator relies on the Protocol Buffer Compiler to orchestrate the
```

Comment:
A link here might be useful (to this site: https://developers.google.com/protocol-buffers/docs/reference/ruby-generated). Your call.

Reply:
Done.